docs/data/data-grid/column-groups/column-groups.md

```diff
@@ -1,18 +1,90 @@
 ---
-title: Data Grid - Column groups 🚧
+title: Data grid - Column groups
 ---
 
-# Data grid - Column groups 🚧
+# Data grid - Column groups
 
 <p class="description">Group your columns.</p>
 
+Grouping columns allows you to have multiple levels of columns in your header and to toggle column groups to show and hide additional columns.
+
+## Define column grouping
+
+You can define the column grouping structure by providing the `columnGroupingModel` prop to the grid.
+This prop receives an array of column groups.
+
+A column group is defined by at least two properties:
+
+- `groupId`: a string used to identify the group
+- `children`: an array containing the children of the group
+
+The `children` attribute can contain two types of objects:
+
+- leafs with type `{ field: string }`, which add the column with the corresponding `field` to this group.
+- other column groups which allows you to have nested groups.
+
+:::warning
+A column can only be associated with one group.
```

Comment:
Maybe we can document what happens when it's used in multiple groups.

Reply:
It throws an error. For now, the field is associated with the first group containing it, but I do not guarantee it will be the case in the future, and the definition of "first group" is not trivial